catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/view/Association.view.js

```diff
@@ -38,8 +38,21 @@ define([
             events: {
                 "click .remove-association": 'removeAssociation'
             },
+            initialize: function(options) {
+                this.readOnly = options.readOnly;
+            },
             removeAssociation: function () {
                 wreqr.vent.trigger('removeAssociation:' + this.model.get('sourceId'), this.model.get('id'));
+            },
+            serializeData: function() {
+                var data = {};
+
+                if (this.model) {
+                    data = this.model.toJSON();
+                }
+                data.readOnly = this.readOnly;
```

Comment:
In this particular case, I think it could have gone in either the view or model. I would have probably opted for the easier approach. Should be fine though.